ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py

```diff
@@ -315,3 +315,61 @@ def test_numpy_negative(
         where=where,
         dtype=dtype,
     )
+
+    
+    
+# floor_divide
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), num_arrays=2,
+        min_value=0,
+        exclude_min=True
+    ),
+    dtype=helpers.get_dtypes("float", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    as_variable=helpers.array_bools(),
+    with_out=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.floor_divide"
+    ),
+    native_array=helpers.array_bools(),
+)
+def test_numpy_floor_divide(
+    dtype_and_x,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    where = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtype,
+        as_variable=as_variable,
+        native_array=native_array,
+    )
+    np_frontend_helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="numpy",
+        fn_tree="floor_divide",
+        x1=np.asarray(x[0], dtype=input_dtype[0]),
```

Comment:
you should pass the input as `x[0]` without converting it into numpy array, `test_frontend_function` was changed recently